sdk/nodejs/helm/v2/helm.ts

```diff
@@ -174,7 +174,12 @@ export class Chart extends yaml.CollectionComponentResource {
                 const yamlStream = execSync(
                     `helm template ${chart} --name ${release} --values ${defaultValues} --values ${values} ${namespaceArg}`
                 ).toString();
-                return this.parseTemplate(yamlStream, cfg.transformations, cfg.resourcePrefix, configDeps);
+                return this.parseTemplate(
+                    yamlStream,
+                    [ignoreHelmTestHook, ...(cfg.transformations || [])],
```

Comment:
Should we make this opt-in or opt-out?

Reply:
We have ~no hope of actually supporting the full behavior of lifecycle hooks, so we should make it opt-in.

Reply:
I could be misreading the whole thin but I am a little worried that you and Luke have different thoughts as to what "opt-in" means here.  

I think Luke's point is "should we be passing this `ignoreHelmTesthook` thing by default or just export `ignoreHelmTesthook` and have folks who need to explicitly pass it in as one of the `cfg.transforms` (i.e. opt-into removing them) and I think you are saying "to a first approximation, if any of these annotations are on the resource, you should have to do something explicit to not have them removed automatically (i.e. opt-into keeping them).

What is unclear to me is in your world how I opt into keeping them.  What does that code look like?

Reply:
I meant: we should get rid of them by default.

I will add a little option in the chartopts that causes us to keep these resources, but I don't think it will be used very rarely.

Reply:
Ok, I've added the option, so you'd do the following. I do think we should pause and make sure that we believe that this is a good thing to add, though. We are not going to be able to support this really well, but it _is_ what we advertise—`helm template` plus Pulumi.

```typescript
new helm.v2.Chart("wordpress", { includeTestHookResources: true })
```